influxdb/client.py

```diff
@@ -64,6 +66,7 @@ def __init__(self,
                  port=8086,
                  username='root',
                  password='root',
+                 pool_size=10,
```

Comment:
I think it makes sense to cast this value to an int, like we do for the `port` parameter in line 81.

Reply:
@xginn8 make sense.